src/tls/openssl/tls_udp.c

```diff
@@ -829,6 +829,20 @@ void dtls_set_mtu(struct dtls_sock *sock, size_t mtu)
 }
 
 
+void dtls_recv_packet(struct dtls_sock *sock, const struct sa *src,
+		      struct mbuf *mb)
+{
+	struct sa addr;
+
+	if (!sock || !src || !mb)
+		return;
+
+	addr = *src;
```

Comment:
I'm curious: Why is the copy needed?

Reply:
the input address is const, but the recv_handler is non-const.

we cannot simply cast away the const-ness, since the recv_handler function
might alter the address.

Reply:
Ah, missed that it's const in the function's signature. Thanks for the explanation. :+1: